src/Components/Artwork/GridItem.tsx

```diff
@@ -125,6 +147,10 @@ const Link = styled(RouterLink)`
   left: 0;
 `
 
+const DisabledLink = styled(Link)`
```

Comment:
`Link`'s are deprecated - we should just use `RouterLink` here, even if it's disabled

Reply:
`Link` here is actually a styled `RouterLink`